<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportManager</class>
 <widget class="QMainWindow" name="ImportManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="createDirectory">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>New Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importFiles">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Import Files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportAll">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Export All</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>To Import/Export files simply use the explorer window &gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Open in Explorer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
